lib/ansible/modules/packaging/os/apt.py

```diff
@@ -206,19 +204,19 @@
     update_cache: yes
     dpkg_options: 'force-confold,force-confdef'
 
-- name: Install a .deb package
+- name: Install a local .deb package
```

Comment:
`local` here can be confusing as most users will think 'the controller', i would rephrase to 'package residing the target file system'